### ISSUE TITLE
2.3.0-rc1 fails to build on iOS

### ISSUE THREAD

**Issue by aMUSiC** [mapsplugin/cordova-plugin-googlemaps#2230]
**I'm submitting a ...**  (check one with "x")
- [ ] question
- [X] any problem or bug report
- [ ] feature request

**If you choose 'problem or bug report', please select OS:**  (check one with "x")
- [ ] **Android**
- [X] **iOS**

When trying to build the app with xcode (9.3) the build fails with errors regarding the StreetView component. On the Android it builds fine.

The last commit I tried which builds fine and works fine on iOS is
https://github.com/mapsplugin/cordova-plugin-googlemaps/commit/611f92d2dd9e8cd13941c8829690924ef82758bb

(As I have to publish the app, I don't have the error output handy. I will provide the error output a bit later)

**Comment by wf9a5m75** [comment]
Please show me the error logs.

**Comment by aMUSiC** [comment]
In PluginStreetViewPanorama.m

line 70:
Interface type cannot be statically allocated
Unknown type name 'GMSPanoramaSource'; did you mean 'GMSPanoramaService'?

line 71:
Use of undeclared identifier 'kGMSPanoramaSourceDefault'
Use of undeclared identifier 'kGMSPanoramaSourceOutside'

line 74 and 76:
No visible @interface for 'GMSPanoramaView' declares the selector 'moveNearCoordinate:radius:source'

**Comment by aMUSiC** [comment]
branch-cordova-sdk 2.6.0 "branch-cordova-sdk"
com.adjust.sdk 4.12.5 "Adjust"
com.googlemaps.ios 2.6.0 "Google Maps SDK for iOS"
com.testfairy.cordova-plugin 2.9.0 "TestFairy Plugin"
cordova-android-play-services-gradle-release 1.3.1 "cordova-android-play-services-gradle-release"
cordova-clipboard 1.1.1 "Clipboard"
cordova-plugin-actionsheet 2.3.3 "ActionSheet"
cordova-plugin-advanced-http 1.11.1 "Advanced HTTP plugin"
cordova-plugin-app-version 0.1.9 "AppVersion"
cordova-plugin-calendar 5.0.0 "Calendar"
cordova-plugin-camera 3.0.0 "Camera"
cordova-plugin-datepicker 0.9.3 "DatePicker"
cordova-plugin-device 1.1.7 "Device"
cordova-plugin-dialogs 2.0.1 "Notification"
cordova-plugin-facebook4 1.10.1 "Facebook Connect"
cordova-plugin-file 6.0.1 "File"
cordova-plugin-firebase 1.0.0 "Google Firebase Plugin"
cordova-plugin-googlemaps 2.3.0-rc1 "cordova-plugin-googlemaps"
cordova-plugin-googleplus 5.3.0 "Google SignIn"
cordova-plugin-inappbrowser 1.7.2 "InAppBrowser"
cordova-plugin-ionic-webview 1.2.0 "cordova-plugin-ionic-webview"
cordova-plugin-network-information 2.0.1 "Network Information"
cordova-plugin-screen-orientation 2.0.2 "Screen Orientation"
cordova-plugin-splashscreen 4.1.0 "Splashscreen"
cordova-plugin-statusbar 2.3.0 "StatusBar"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-plugin-x-socialsharing 5.3.2 "SocialSharing"
cordova-sqlite-storage 2.3.0 "Cordova sqlite storage plugin"
es6-promise-plugin 4.1.0 "Promise"
ionic-plugin-keyboard 2.2.1 "Keyboard"
uk.co.workingedge.phonegap.plugin.launchnavigator 4.2.0 "Launch Navigator"

I'll see what I can do about the project files tomorrow

**Comment by wf9a5m75** [comment]
I have no problem.

```
$> cordova plugin list
com.googlemaps.ios 2.6.0 "Google Maps SDK for iOS"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-googlemaps 2.3.0-rc1 "cordova-plugin-googlemaps"
cordova-plugin-ionic-keyboard 2.0.5 "cordova-plugin-ionic-keyboard"
cordova-plugin-ionic-webview 1.2.0 "cordova-plugin-ionic-webview"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-whitelist 1.3.3 "Whitelist"
```
![screen shot 2018-04-20 at 12 18 37 pm](https://user-images.githubusercontent.com/167831/39069565-ff894960-4494-11e8-9a6e-bbaef307e567.png)

**Comment by wf9a5m75** [comment]
ping

**Comment by wf9a5m75** [comment]
Since there is no response, I close this thread.

**Comment by aMUSiC** [comment]
Apologies for not providing more information, I got really busy with work and didn't have the time to create a project that demonstrates this.

However, the issue has been fixed.

Before installing the rc-1 version of multiple_maps, I had to remove the com.googlemaps.ios plugin too. Once I did that and the plugin was refetched as a dependency of cordova.plugin.googlemaps, it built successfuly on ios as well.

Thank you for your time.
